_release/org.cacophony.Attiny.conf

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?> <!-- -*- XML -*- -->
+
+<!DOCTYPE busconfig PUBLIC
+ "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN"
+ "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
+<busconfig>
+  <policy user="root">
+    <allow own="org.cacophony.Attiny"/>
```

Comment:
Any reason not to call the service "ATtiny" to match the actual device name? 
(I know I'm being picky)